packages/tinacms/src/toolkit/fields/plugins/group-list-field-plugin.tsx

```diff
@@ -150,13 +150,13 @@ const Item = ({
   const removeItem = React.useCallback(() => {
     tinaForm.mutators.remove(field.name, index)
   }, [tinaForm, field, index])
-  const title = label || (field.label || field.name) + ' Item'
+  const title = label || `${field.label || field.name} Item`
 
   const { dispatch: setHoveredField } = useEvent<FieldHoverEvent>('field:hover')
   const { dispatch: setFocusedField } = useEvent<FieldFocusEvent>('field:focus')
   return (
     <Draggable
-      type={field.name}
+      // type={field.name}
```

Comment:
**question**: Why is this commented-out code retained?

Reply:
removed